xilem/examples/gallery_label_xilem_view.rs

```diff
@@ -0,0 +1,178 @@
+use winit::error::EventLoopError;
+use xilem::view::{
+    flex, grid, label, label as l, portal, prose, sized_box, Axis, GridExt, Label, Padding, Prose,
+};
+use xilem::{palette::css, EventLoop, FontWeight, LineBreaking, TextAlignment, WidgetView, Xilem};
+const LABEL_COLOR: Color = css::ROYAL_BLUE;
+
+fn title_prose(text: impl Into<ArcStr>) -> Prose {
+    prose(text)
+        .text_size(18.0)
+        .alignment(TextAlignment::Justified)
+        .brush(css::GREEN)
+}
+fn txt_prose(text: impl Into<ArcStr>) -> Prose {
+    prose(text)
+        .text_size(14.0)
+        .alignment(TextAlignment::Start)
+        .brush(Color::from_rgb8(0x11, 0x11, 0x11))
+}
+fn lc(text: impl Into<ArcStr>) -> Label {
+    //colored label
+    label(text).brush(LABEL_COLOR)
+}
+fn app_logic(_d: &mut AppState) -> impl WidgetView<AppState> {
+    let m_c = Color::from_rgb8(0x11, 0x11, 0x11); //main text
+    let l_c = LABEL_COLOR;
+    let mut i = 1;
+
+    portal(
+    flex((
+    (txt_prose("Xilem view::Label formats vGit@25-02 #25b12ad (in a ↕-scrollable area)").text_size(18.0),
+    if cfg!(debug_assertions) {txt_prose(
+     "This is a debug build, so you can use github.com/linebender/xilem/tree/main/masonry#debugging-features:
+        • F11 to toggle a rudimentary widget inspector
+        • F12 to toggle paint widget layout rectangles")
+    } else {txt_prose("This is not a debug build, so F11 widget inspector and F12 widget rectangles tools are not available)\ngithub.com/linebender/xilem/tree/main/masonry#debugging-features")},
+    label(format!("Label: Serif Bold 14 {LABEL_COLOR:?}")).text_size(14.0).weight(FontWeight::BOLD) // float bold=700, FontWeight::parse("normal") for css
+        .font(fontique::GenericFamily::Serif)
+        .alignment(TextAlignment::Start)
+        .brush(l_c) //impl Into<peniko:Brush> brush sets text color, but gradients and images are also supported Solid(color::AlphaColor<Srgb>) Gradient(Gradient) Image(Image),
+        .line_break_mode(LineBreaking::Overflow) //WordWrap Clip Overflow
+        ,
+    title_prose(format!("§{i} .alignment")),{i+=1;},
+    txt_prose("  4 options: ≝Start Middle End Justified\n  https://docs.rs/parley/latest/parley/layout/enum.Alignment.html")
+    ),
+    // doesn't seem to be different now vs unconstrained
+    // (lc("  •flex in a 200×70 box to show impact of constraints ").alignment(TextAlignment::Start),
+    // sized_box(
+    //   flex((
+    //     lc("1/4 alignment Start"     ).alignment(TextAlignment::Start        ),
+    //     lc("2/4 alignment Middle"    ).alignment(TextAlignment::Middle       ),
+    //     lc("3/4 alignment End"       ).alignment(TextAlignment::End          ),
+    //     lc("4/4 alignment Justified" ).alignment(TextAlignment::Justified    ),
+    //   ))
+    //   ).width(200f64).height(70f64).padding(Padding::from(0.))
+    //    .background(css::LIGHT_GRAY) // .border(css::RED,0.).rounded(RoundedRectRadii::from_single_radius(0.))
+    // ,),
```

Comment:
This will need to be resolved before we can merge; it's not clear what it's doing

Reply:
repurposed this into a comparison of how labels spread within a flex vs grid